tests/elk_entities/test_nodes.py

```diff
@@ -14,19 +14,29 @@ def test_get_nodes_from_elastic(self):
         node_registry = NodeRegistry()
         node_registry.get_nodes_from_elastic(self.elk_client)
         self.assertEqual(1, len(node_registry.nodes))
-        self.assertIn('E43TT5r9Q9-VWckuAXLf0Q', node_registry.nodes)
-        node = node_registry.nodes['E43TT5r9Q9-VWckuAXLf0Q']
-        self.assertEqual('E43TT5r9Q9-VWckuAXLf0Q', node.node_id)
-        self.assertEqual('1.1.1.1:9200', node.publish_address)
-        self.assertEqual(('master', 'data', 'ingest'), node.node_type)
+        self.assertIn("E43TT5r9Q9-VWckuAXLf0Q", node_registry.nodes)
+        node = node_registry.nodes["E43TT5r9Q9-VWckuAXLf0Q"]
+        self.assertEqual("E43TT5r9Q9-VWckuAXLf0Q", node.node_id)
+        self.assertEqual("1.1.1.1:9200", node.publish_address)
+        self.assertEqual(("master", "data", "ingest"), node.node_type)
+
```

Comment:
Can you add a test here? You can use the test above and below for examples. You should call `get_nodes_from_elastic` with `wan_only=True`. You should expecgt the node's publish address to be `None` so you could check something like
```python
self.assertIsNone(node.publish_address)
```
All the other assertions should probably be the same.